modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -0,0 +1,116 @@
+[[ssl-certs-imported]]
+= Imported SSL Certificates
+
+By default, {productname} uses a self-signed certificate.
+For additional security, you can import a custom certificate, signed by a third party certificate authority (CA).
+
+This section covers how to use an imported SSL certificate with a new {productname} installation, and how to replace existing self-signed certificates with imported certificates.
+
+Before you begin, ensure you have:
+
+* A certificate authority (CA) SSL public certificate
+* An SSL server key
+* An SSL server certificate
+
+Your key and certificate files must be in PEM format.
+
+The host name of the SSL keys and certificates must match the fully qualified host name of the machine you deploy them on.
+You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
+You can also list multiple host names if your environment requires it.
+
+
+== Imported Certificates for New Installations
+
+By default, {productname} uses a self-signed certificate.
+After you have completed the initial setup, you can replace the default certificate with an imported certificate.
+
+
+
+.Procedure: Installing Imported Certificates on a New {productname} Server
+
+. Install the {productname} Server according to the instructions in xref:installation:install-intro.adoc[].
+. Complete the initial setup according to xref:installation:server-setup.adoc[].
+. At the command prompt, point the SSL environment variables to the imported certificate file locations:
++
+----
+export CA_CERT=<path_to_CA_certificate_file>
+export SERVER_KEY=<path_to_web_server_key>
+export SERVER_CERT=<path_to_web_server_certificate>
+----
+. Complete {productname} setup:
++
+----
+yast susemanager_setup
+----
++
+When you are prompted for certificate details during setup, fill in random values.
+The values will be overridden by the values you specified at the command prompt.
+
+[NOTE]
+====
+Execute the [command]``yast susemanager_setup`` command from the same shell you exported the environment variables from.
+====
+
+
+== Imported Certificates for New Proxy Installations
+
+By default, {productname} Proxy uses a self-signed certificate.
+After you have completed the initial setup, you can replace the default certificate with an imported certificate.
+
+.Procedure: Installing Imported Certificates on a New {productname} Proxy
+
+. Install the {productname} Proxy according to the instructions in xref:installation:install-intro.adoc[].
+. Complete the initial setup according to xref:installation:proxy-setup.adoc[].
+. At the command prompt, run:
++
+----
+configure-proxy.sh
+----
+. At the ``Do you want to import existing certificates?`` prompt, type kbd:[y].
+. Follow the prompts to complete setup.
+
```

Comment:
A good place to add a Note that all server certificates for server and proxies must be singed by the same CA.